openc3-cosmos-init/plugins/packages/openc3-cosmos-ace-diff/src/index.js

```diff
@@ -368,10 +375,11 @@ function copy(acediff, e, dir) {
   let contentToInsert = ''
   for (let i = startLine; i < endLine; i += 1) {
     contentToInsert += getLine(sourceEditor, i)
-    // If the first line is blank we add an extra newline
+    // JMT: If the first line is blank we add an extra newline
     // because when it is inserted the first one is effectively eaten
     // by the replace and merging of the lines
-    if (i === startLine && contentToInsert === '') {
+    // However if the targetStartLine is the very top then we don't
+    if (i === startLine && contentToInsert === '' && targetStartLine != 0) {
       contentToInsert += '\n'
```

Comment:
I think I added this a while ago and while it's needed in some cases it doesn't work if inserting at the very top.